src/lib/model/OutboundTransfersModel.js

```diff
@@ -208,30 +208,32 @@ class OutboundTransfersModel {
                     });
 
                     // check we got the right payee and info we need
-                    if(payee.partyIdInfo.partyIdType !== this.data.to.idType) {
+                    if (payee.partyIdInfo.partyIdType !== this.data.to.idType) {
                         const err = new Error(`Expecting resolved payee party IdType to be ${this.data.to.idType} but got ${payee.partyIdInfo.partyIdType}`);
                         return reject(err);
                     }
 
-                    if(payee.partyIdInfo.partyIdentifier !== this.data.to.idValue) {
+                    if (payee.partyIdInfo.partyIdentifier !== this.data.to.idValue) {
                         const err = new Error(`Expecting resolved payee party identifier to be ${this.data.to.idValue} but got ${payee.partyIdInfo.partyIdentifier}`);
                         return reject(err);
                     }
 
-                    if(payee.partyIdInfo.partySubIdOrType !== this.data.to.idSubValue) {
+                    if (payee.partyIdInfo.partySubIdOrType !== this.data.to.idSubValue) {
                         const err = new Error(`Expecting resolved payee party subTypeId to be ${this.data.to.idSubValue} but got ${payee.partyIdInfo.partySubIdOrType}`);
                         return reject(err);
                     }
 
-                    if(!payee.partyIdInfo.fspId) {
+                    if (!payee.partyIdInfo.fspId) {
```

Comment:
why?

Reply:
Not sure if it is the IDE settings that changed them . I see further down in the code some more "IFs" with a spaces and then up at line 112 "IFs" without spaces. Ill remove the spaces between the If and the condition throughout.

<img width="1139" alt="Screenshot 2020-04-03 at 11 20 18" src="https://user-images.githubusercontent.com/13884584/78346570-7c119b80-759f-11ea-94ea-99c082eff021.png">